sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOrcScan.scala

```diff
@@ -1772,6 +1904,32 @@ class MultiFileCloudOrcPartitionReader(
    */
   override def getFileFormatShortName: String = "ORC"
 
+  override def canUseCombine: Boolean = {
+    // FIXME whether should "ignoreCorruptFiles" be considered here? Since coalescing read does.
```

Comment:
The combining reader isn't affected by ignorecorruptFiles,  the multi-file reader handles it, you get a HostMemoryEmptyMetaData back before combining takes place and the combining handles those.

Reply:
got it, thx.